src/style.css

```diff
@@ -0,0 +1,130 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc) format("truetype");
+}
+
+@font-face {
+  font-family: "Avenir-book";
+  src: url(./fonts/Avenir-Book.ttf) format("truetype");
+}
+
+@font-face {
+  font-family: "Avenir-heavy";
+  src: url(./fonts/Avenir-Heavy.ttf) format("truetype");
+}
+
+body {
+  font-family: "Avenir", Helvetica, sans-serif;
+  font-weight: 300;
+  margin: 0 8px;
+}
+
+.container {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.search-bar {
+  width: 100%;
+  max-height: 70px;
+  margin-top: 20px;
+  display: flex;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  line-height: 22px;
+  border-radius: 4px;
+}
+
+.search-input-big {
+  width: 100%;
+  border: 0;
+  outline: none;
+  font-family: "Avenir", Helvetica, sans-serif;
+  box-shadow: 0 0;
+  font-size: 16px;
+  border-radius: 4px;
+}
+
+.serchbo {
+  padding: 0;
+}
+
+.search-button-big {
+  margin: 0;
+  padding: 25px 17px 26px 26px;
+}
+
+.search-bar-big button img {
+  height: 19px;
+  width: 19px;
+  align-items: center;
+  left: 26px;
+  top: 25px;
+}
+
+.search-bar-big button {
+  border: 0;
+  left: 26px;
+  top: 25px;
+  background: #fff;
+  cursor: pointer;
+  border-radius: 4px;
+}
```

Comment:
Do not use tag selctor and don't increase selectors specificity 
![2023-02-13_15h55_37](https://user-images.githubusercontent.com/61904995/218476605-06db4e2d-6a7e-4e62-8bad-026c649faa39.png)